bridge_adaptivity/bridge_lti/consumer.py

```diff
@@ -71,6 +71,8 @@ def source_preview(request):
         # student flow
         sequence_item = SequenceItem.objects.get(id=sequence_item_id)
         activity = sequence_item.activity
+        if activity.repetition > 1:
```

Comment:
Can you move this logic into getters/setters of the model for localizing it?

Reply:
done